src/ansible_navigator/image_manager/puller.py

```diff
@@ -142,18 +151,33 @@ def prologue_stdout(self):
         print("Updating the execution environment")
         print("\u002d" * width)
 
+    def _generate_pull_command(self) -> List:
+        """Generate the pull command.
+
+        :returns: The list of command parts
+        """
+        command_line = [self._container_engine, "pull"]
+        if isinstance(self._arguments, list):
+            for argument in self._arguments:
+                command_line.extend(shlex.split(argument))
```

Comment:
`+=` might be more conventional than `extend()`.

Reply:
I switched to extend a while back and I don't recall why. Might be because of `append`.  

For the sake of research some interesting differences here which I was unaware of: https://stackoverflow.com/questions/3653298/concatenating-two-lists-difference-between-and-extend

This one I'll leave as is.

Reply:
@relrod Decided to keep it extend, resolving this